lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -424,6 +424,22 @@ class CategoryRenderer {
     }
   }
 
+  /**
+   * @param {LH.ReportResult.Category} category
+   * @param {Object<string, LH.Result.ReportGroup>} [groupDefinitions]
+   * @return {Element}
+   */
+  render(category, groupDefinitions = {}) {
```

Comment:
(the diff is a little funny here because i moved the diagram... but the method signature and top 3 lines are identical to what it had been.)


<details>
<summary>git diff catches the changes as a "move", and calls out the new method of renderClumps a bit more clearer</summary>

![image](https://user-images.githubusercontent.com/39191/93545190-ae784080-f914-11ea-977d-c7c896760e6e.png)


</details>